cmake/Findsndfile.cmake

```diff
@@ -0,0 +1,32 @@
+include(FindPackageHandleStandardArgs)
```

Comment:
Please rename this file to `FindSndFile.cmake` to make it compatible with the upstream [`SndFileConfig.cmake`](https://github.com/libsndfile/libsndfile/blob/master/cmake/SndFileConfig.cmake.in).
Make sure to create the imported unknown CMake target `SndFile::sndfile`.